gradle.properties

```diff
@@ -8,17 +8,20 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 ### dependencies
-pravegaVersion=0.4.0
-flinkConnectorVersion=0.4.0
+pravegaVersion=0.5.0-2223.ce3ba81-SNAPSHOT
+flinkConnectorVersion=0.5.0-75.01ba05e-SNAPSHOT
 
 ### Pravega-samples output library
 samplesVersion=0.5.0-SNAPSHOT
 
 ### Flink-connector examples
-flinkVersion=1.6.0
+flinkVersion=1.7.2
+flinkScalaVersion=2.12
 
 #hadoop connector
 hadoopVersion=2.8.1
-scalaVersion=2.11.8
 sparkVersion=2.2.0
-hadoopConnectorVersion=0.4.0
+hadoopConnectorVersion=0.5.0-29.d6fa817-SNAPSHOT
+
+scalaJava8CompatVersion=0.9.0
```

Comment:
Maybe we could better organize the items in this file with proper title and maybe some comments. For instance, it is confusing that we have 3 different scala-related dependencies, so a comment may clarify the picture.

Reply:
re-arranged variables and added comments